### ISSUE TITLE
what is relation of  spare finetuning with the spasegpt mentioned in paper? 

### ISSUE THREAD

**Issue by SHUSHENGQIGUI** [IST-DASLab/SparseFinetuning#3]
## Additional context
![image](https://github.com/IST-DASLab/SparseFinetuning/assets/28890775/21f31bb2-2020-447c-ae25-70bdfffc3bce)

**Comment by eldarkurtic** [comment]
SparseGPT is an algorithm we used to prune models to desired sparsity levels.
Sparse Fine-Tuning is an approach to fine-tune that model to get better accuracy on the target dataset.

**Comment by SHUSHENGQIGUI** [comment]
> SparseGPT is an algorithm we used to prune models to desired sparsity levels. Sparse Fine-Tuning is an approach to fine-tune that model to get better accuracy on the target dataset.

Thanks for replying on me . could you show the accuarcy comparison of pruned model via sparsegpt and  that model after Sparse Fine-Tuning?  Thank you

**Comment by eldarkurtic** [comment]
| sparsity | SparseGPT | SparseGPT + Sparse Finetuning |
|:--------:|:---------:|:-----------------------------:|
| 50%      |   23.2    |            30.6               |
| 60%      |   18.3    |            28.8               |
| 70%      |   14.3    |            28.0               |
| 80%      |    9.5    |            23.1               |

**Comment by SHUSHENGQIGUI** [comment]
> sparsity	SparseGPT	SparseGPT + Sparse Finetuning
> 50%	23.2	30.6
> 60%	18.3	28.8
> 70%	14.3	28.0
> 80%	9.5	23.1

Thanks very much. but  which model and which metric above?

**Comment by eldarkurtic** [comment]
The ones that we present in the paper: MPT-7b model finetuned on the GSM8k dataset. 
The metric is 0-shot accuracy evaluated through lm-evaluation-harness.
